File: crates/godlint-cli/tests/fixtures/rules/else-if-chain/chain.ts
function accepted(value: number): number {
  if (value === 1) {
    return 10;
  } else if (value === 2) {
    return 20;
  } else if (value === 3) {
    return 30;
  }

  return 0;
}

function reported(value: number) {
  if (value === 1) {
    for (const item of items) {
      work(item);
    }
  }
}

File: crates/godlint-cli/tests/fixtures/rules/else-if-chain/chain.rs
fn accepted(value: u32) -> u32 {
    if value == 1 {
        10
    } else if value == 2 {
        20
    } else if value == 3 {
        30
    } else {
        0
    }
}

fn reported(value: u32) {
    if value == 1 {
        for item in items {
            work(item);
        }
    }
}
